cmake/external/eigen.cmake

```diff
@@ -20,6 +20,8 @@ set(EIGEN_SOURCE_DIR ${THIRD_PARTY_PATH}/eigen3/src/extern_eigen3)
 set(EIGEN_REPOSITORY https://gitlab.com/libeigen/eigen.git)
 set(EIGEN_TAG        f612df273689a19d25b45ca4f8269463207c4fee)
 
+file(WRITE ${PADDLE_SOURCE_DIR}/paddle/pten/api/include/eigen_include.h "#include \"${EIGEN_SOURCE_DIR}/Eigen/Core\"")
+
```

Comment:
Is it possible to copy only bfloat16 header related file into this location ?

Reply:
No. The bfloat16 implementation of eigen requires a big part of the eigen library to work and trying to copy more files can cause issues as mentioned above.